scss/tech.scss

```diff
@@ -0,0 +1,83 @@
+.cont-tech {
+  display: flex;
+  height: 400px;
+  justify-content: flex-start;
+  flex-wrap: wrap;
+  margin: 40px auto;
+  max-width: 1000px;
+  width: 100%;
+}
+
+.technology, .technology-large {
+  background-color: #BED23C;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  height: 250px;
+  margin: 30px 10px 0px 30px;
+  position: relative;
+  width: 190px;
+
+    &::before {
+      border-radius: 5px;
+      border-style: solid;
+      border-color: #BED23C;
```

Comment:
You can use the shorthand: `border: 1px solid <color>;` (Also remember to use a variable for the color :smile: )